WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsLearnMore.swift

```diff
@@ -50,8 +68,13 @@ private enum Localization {
         comment: "Generic error message when In-Person Payments is unavailable"
     )
 
-    static let learnMore = NSLocalizedString(
-        "Tap to learn more about accepting payments with your mobile device and ordering card readers",
+    static let learnMoreLink = NSLocalizedString(
+        "Learn more",
+        comment: "A label prompting users to learn more about card readers. This part is the link to the website"
+    )
+
+    static let learnMoreText = NSLocalizedString(
+        "about accepting payments with your mobile device and ordering card readers",
```

Comment:
The localization of concatenated strings like this seems like it could be difficult in some languages. With this approach, the translator is required to start the sentence with a call to action.

You could try using [positional placeholders](https://github.com/woocommerce/woocommerce-ios/blob/develop/docs/localization.md#always-use-positional-placeholders) for this, and reference that the `learnMoreLink` will be in a placeholder in `learnMoreText`. I'm not exactly sure what changes you'd need to make to the `AttributedString` creation though, you'd have to find the range of the translated `learnMoreLink` in the combined string and apply attributes to that.

Reply:
Good point! Improved here - https://github.com/woocommerce/woocommerce-ios/pull/5513/commits/cef2d7c1bfbf0f127ca7e656896d5128c230cd71